src/crypto/tls/handshake_client_tls13.go

```diff
@@ -450,6 +455,9 @@ func (hs *clientHandshakeStateTLS13) readServerParameters() error {
 		c.sendAlert(alertUnexpectedMessage)
 		return unexpectedMessageError(encryptedExtensions, msg)
 	}
+
+	hs.handshakeTimings.ReadEncryptedExtensions = hs.handshakeTimings.elapsedTime()
```

Comment:
I'd bump this to the end of the function so that this duration accounts for the time it takes to process the inputs (and not just read them). Here and below.

Reply:
Good call.